sandbox/settings.py

```diff
@@ -227,41 +227,56 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
 
     SOCIAL_ERROR_REVERSE_ID = values.Value()
 
+    # AUTHENTICATION
+    AUTHENTICATION_BACKEND = {
+        "BASE_URL": values.Value(
+            "", environ_name="AUTHENTICATION_BASE_URL", environ_prefix=None
+        ),
+        "BACKEND": values.Value(
```

Comment:
There is still a LMSHandler interface. That has been moved on frontend. But if someone want to implement a interface to be compatible with Moodle API (to login user and enroll to course), it can implement it and to use it we just have to update this BACKEND property.

Currently, we can use two different API interfaces:
- Demo: which fakes enrollment requests (we use it with http)
- Edx: which makes real requests to Edx api but require https.

I kept old backends names just for convenience.